gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -343,7 +343,7 @@
                   <col class="col-xs-2"/>
                 </colgroup>
                 <tr ng-repeat="entry in currentFolder.xattrFilter.entries">
-                  <td>
+                  <td class="extended-attributes-filter-rule-checkbox">
                     <input type="checkbox" ng-model="entry.permit">
```

Comment:
How about putting the style on the checkbox directly?

```suggestion
                    <input class="extended-attributes-filter-rule-checkbox" type="checkbox" ng-model="entry.permit">
```

This would simplify the CSS even further.

Reply:
Done in 3eb382a75862d790513a65de322595686a348140.